src/common/FilePreviewCommon/monacoSpecialLanguages.js

```diff
@@ -6,11 +6,9 @@ import { gitignoreDefinition } from './customLanguages/gitignore.js';
 export async function registerAdditionalLanguages(monaco){
     await languageDefinitions();
     registerAdditionalLanguage("cppExt", [".ino", ".pde"], "cpp", monaco)
-    registerAdditionalLanguage("xmlExt", [".xsd", ".wsdl", ".xslt", ".csproj", ".vcxproj", ".vbproj", ".fsproj"], "xml", monaco)
-    registerAdditionalLanguage("jsonExt", [".vsconfig"], "json", monaco)
-    registerAdditionalLanguage("txtExt", [".sln"], "txt", monaco)
+    registerAdditionalLanguage("xmlExt", [".wsdl", ".csproj", ".vcxproj", ".vbproj", ".fsproj"], "xml", monaco)
+    registerAdditionalLanguage("txtExt", [".sln", ".log", ".vsconfig", ".json"], "txt", monaco)
```

Comment:
why is `.json` under `txt` ? Is it no longer supported by default?

Reply:
yeah, I think you can remove json from here. Other than that everything looks fine. Nice work!